### ISSUE TITLE
Add support for multiple raster overlays on terrain

### ISSUE THREAD

**Issue by baothientran** [CesiumGS/cesium-unreal#349]
Currently only one raster is draped over terrain. We should allow multiple layers to drape over terrain

**Comment by liujingjie** [comment]
1、i changed the `GltfMaterialWithOverlays` and add another 3 ,and rename them as OverlayRcct4、OverlayRcct5、OverlayRcct6 like this 
![image](https://user-images.githubusercontent.com/9113753/120259089-ddc5b680-c2c5-11eb-923c-d3cd9c5b36f0.png)

2、 i add two rasterlayer in the Cesium World Terrain, and then i get a result 
![image](https://user-images.githubusercontent.com/9113753/120259276-3c8b3000-c2c6-11eb-8cdb-82287eb0c360.png)

it seems like i can add multiple raster overlays on terrain,but i  want to konw why i should add  3 RasterOverlay,and in the code `CesiumGltfComponent.cpp` 
```
void UCesiumGltfComponent::AttachRasterTile(
    const Cesium3DTiles::Tile& tile,
    const Cesium3DTiles::RasterOverlayTile& rasterTile,
    UTexture2D* pTexture,
    const CesiumGeometry::Rectangle& textureCoordinateRectangle,
    const glm::dvec2& translation,
    const glm::dvec2& scale) {
  if (this->OverlayTiles.Num() == 0) {
    // First overlay tile, generate texture coordinates
    // TODO
  }

  this->OverlayTiles.Add(FRasterOverlayTile{
      pTexture,
      FLinearColor(
          textureCoordinateRectangle.minimumX,
          textureCoordinateRectangle.minimumY,
          textureCoordinateRectangle.maximumX,
          textureCoordinateRectangle.maximumY),
      FLinearColor(translation.x, translation.y, scale.x, scale.y)});

  if (this->OverlayTiles.Num() > 3) {
    UE_LOG(LogCesium, Warning, TEXT("Too many raster overlays"));
  }

  this->UpdateRasterOverlays();
}
```
why OverlayTiles`s length has 3 limit.

**Comment by kring** [comment]
@liujingjie the code has a limit of 3 because the Material does. If you've edited the material to allow more, you should edit the code as well. It'd be a nice improvement to make the code query the material so it can work with whatever number of overlays are available.

**Comment by liujingjie** [comment]
![image](https://user-images.githubusercontent.com/9113753/121004302-19c6b300-c7c1-11eb-8f1c-e6de880340a1.png)
When two layers are loaded, will the second layer always load tiles? When I load 2 layers, every time I move the camera, the second layer will be reloaded.Where I should watch to  solve this problem?  
2、 why in the `CesiumGltfWithOverlays` there was 3 CesiumRasterOverlay by default, i remove one or two ,then the rasterlay miss some tiles 。Is there any reason for the design like this？

**Comment by kring** [comment]
Fixed in #608.
